docs/pip/packages.md

```diff
@@ -107,18 +107,32 @@ Install from a `pyproject.toml` file with all optional dependencies enabled:
 $ uv pip install -r pyproject.toml --all-extras
 ```
 
-To install dependency groups in the current project directory's pyproject.toml:
+To install dependency groups in the current project directory's `pyproject.toml`, for example the
+group `foo`:
 
 ```console
 $ uv pip install --group foo
 ```
 
-To install dependency groups for an arbitrary pyproject.toml:
+To specify the project directory where groups should be sourced from:
 
 ```console
-$ uv pip install --group some/path/pyproject.toml:foo
+$ uv pip install --project some/path/ --group foo --group bar
 ```
 
+Alternatively, you can specify a path to a `pyproject.toml` for each group:
+
+```console
+$ uv pip install --group some/path/pyproject.toml:foo --group other/pyproject.toml:bar
+```
+
+!!! note
+
+    `--group` flags do not apply to other sources specified with flags like `-r` or -e`.
```

Comment:
You could say, "As in pip, " if you want to mention that it is done to match pip's behavior.

Reply:
Ooh I like that. I felt the need to specify that because I don't think we would have picked that behaviour in a vacuum and I expect some people will want to suggest changing it. Maybe too defensive 😅